BurnOutSharp/ProtectionType/SafeDisc.cs

```diff
@@ -9,8 +9,20 @@
 
 namespace BurnOutSharp.ProtectionType
 {
-    // TODO: Figure out how to properly distinguish SafeDisc and SafeCast since both use
-    // the same generic BoG_ string. The current combination check doesn't seem consistent
+    /// <summary>
+    /// SafeDisc was an [INSERT REST OF DOCS HERE]
```

Comment:
The comment denotes incompletion. Are the docs already inserted?

Reply:
Deliberately incomplete as more notes are coming